src/cdk-experimental/menu/menu-item-selectable.ts

```diff
@@ -6,25 +6,23 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {coerceBooleanProperty, BooleanInput} from '@angular/cdk/coercion';
-import {Input, Directive, Output, EventEmitter} from '@angular/core';
+import {BooleanInput, coerceBooleanProperty} from '@angular/cdk/coercion';
+import {Directive, Input} from '@angular/core';
 import {CdkMenuItem} from './menu-item';
 
-/** Counter used to set a unique id and name for a selectable item */
-let nextId = 0;
-
 /**
  * Base class providing checked state for MenuItems along with outputting a clicked event when the
  * element is triggered. It provides functionality for selectable elements.
  */
-@Directive()
+@Directive({
+  host: {
+    '[attr.aria-checked]': 'checked || null',
```

Comment:
Should this one be just `[attr.aria-checked]: 'checked'`? I'm not 100% sure if removing it completely is going be read out as "not checked" by the screen reader.

Reply:
That's a good point. I just moved these lines from somewhere else, but I think they should be true or false